package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@automattic/vip",
-  "version": "2.39.7-dev.0",
+  "version": "2.40.0-dev.2",
```

Comment:
Please don't do this. The [prepare release workflow](https://github.com/Automattic/vip-cli/actions/workflows/npm-prepare-release.yml) updates the package version.